orca_detector/vggish_params.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 The TensorFlow Authors All Rights Reserved.
```

Comment:
We may not actually need this file any more, now that I've split off `mel_params.py`.  When I make my other changes tomorrow, I'll confirm and delete if I can.

Reply:
removed